Cargo.toml

```diff
@@ -18,15 +18,24 @@ colored = "2.0.0"
 ion-rs = "0.18.1"
 memmap = "0.7.0"
 tempfile = "3.2.0"
-ion-schema = "0.7.0"
+ion-schema = "0.10.0"
+serde = { version = "1.0.163", features = ["derive"] }
 serde_json = { version = "1.0.81", features = [ "arbitrary_precision", "preserve_order" ] }
 base64 = "0.21.1"
+tera = "1.18.1"
+convert_case = "0.6.0"
+matches = "0.1.10"
+thiserror = "1.0.50"
 
 [dev-dependencies]
 rstest = "~0.17.0"
 assert_cmd = "~1.0.5"
 tempfile = "~3.5.0"
 
+[features]
+deafault = []
+beta-subcommands = []
```

Comment:
🗺️ Adds a `beta-subcommands` feature for all beta subcommands (https://github.com/amazon-ion/ion-cli/issues/63)